modules/apps/layout/layout-test/src/testIntegration/java/com/liferay/layout/change/tracking/test/LayoutClassedModelUsageTableReferenceDefinitionTest.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.layout.change.tracking.test;
+
+import com.liferay.arquillian.extension.junit.bridge.junit.Arquillian;
+import com.liferay.change.tracking.test.util.BaseTableReferenceDefinitionTestCase;
+import com.liferay.layout.model.LayoutClassedModelUsage;
+import com.liferay.layout.service.LayoutClassedModelUsageLocalService;
+import com.liferay.portal.kernel.model.change.tracking.CTModel;
+import com.liferay.portal.kernel.test.rule.AggregateTestRule;
+import com.liferay.portal.kernel.test.util.RandomTestUtil;
+import com.liferay.portal.test.rule.Inject;
+import com.liferay.portal.test.rule.LiferayIntegrationTestRule;
+import com.liferay.portal.test.rule.PermissionCheckerMethodTestRule;
+
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.runner.RunWith;
+
+/**
+ * @author Gislayne Vitorino
+ */
+@RunWith(Arquillian.class)
+public class LayoutClassedModelUsageTableReferenceDefinitionTest
+	extends BaseTableReferenceDefinitionTestCase {
+
+	@ClassRule
+	@Rule
+	public static final AggregateTestRule aggregateTestRule =
+		new AggregateTestRule(
+			new LiferayIntegrationTestRule(),
+			PermissionCheckerMethodTestRule.INSTANCE);
+
+	@Before
+	@Override
+	public void setUp() throws Exception {
+		_layoutClassedModelUsage =
+			_layoutClassedModelUsageLocalService.createLayoutClassedModelUsage(
+				RandomTestUtil.randomLong());
+	}
+
+	@Override
+	protected CTModel<?> addCTModel() throws Exception {
+		return _layoutClassedModelUsageLocalService.addLayoutClassedModelUsage(
+			_layoutClassedModelUsage);
```

Comment:
This add method takes in an object and add it to the database, but since the object you created only has the primary key, it's not useful. You should use [addLayoutClassedModelUsage](https://github.com/liferay/liferay-portal/blob/master/modules/apps/layout/layout-service/src/main/java/com/liferay/layout/service/impl/LayoutClassedModelUsageLocalServiceImpl.java#L37-L73) to add the entity. See `LayoutCopyHelperTest` for example.